src/bundle/Resources/public/js/scripts/admin.input.text.js

```diff
@@ -39,8 +39,31 @@
         inputClearBtns.forEach((clearBtn) => clearBtn.addEventListener('click', clearText, false));
         passwordTogglerBtns.forEach((passwordTogglerBtn) => passwordTogglerBtn.addEventListener('click', togglePasswordVisibility, false));
     };
+    const handleInputChange = ({ target: { value } }, btn) => {
+        btn.disabled = value === '';
+    };
+    const initExtraBtns = (event) => {
+        const extraBtns = doc.querySelectorAll('.ibexa-input-text-wrapper__action-btn--extra-btn');
+
+        extraBtns.forEach((btn) => {
+            const input = btn.closest('.ibexa-input-text-wrapper').querySelector('input');
+
+            if (!input) {
+                return;
+            }
+
+            const marginClearButton = 5;
```

Comment:
Move `marginClearButton` and `marginWidth` to second line under `(function (global, doc) { ` and I would change variables name 
`const MARGIN_CLEAR_BUTTON = 5;`
`const MARGIN_WIDTH_BUTTION = 24;`
?